drivers/hmis/app/graphql/mutations/base_mutation.rb

```diff
@@ -120,7 +88,7 @@ def default_delete_record(record:, field_name:)
       if record.present?
         record.destroy
       else
-        errors << InputValidationError.new("#{field_name.to_s.humanize} record not found", attribute: 'id') unless record.present?
+        errors << HmisErrors::CustomValidationError.new(field_name, :not_found) unless record.present?
```

Comment:
given that this is in an else for `record.present?` I don't think you need the extra check.
```suggestion
        errors << HmisErrors::CustomValidationError.new(field_name, :not_found) 
```